odoo/addons/base/res/res_partner_view.xml

```diff
@@ -342,9 +343,9 @@
                    <field name="user_id"/>
                    <field name="parent_id" domain="[('is_company','=',1)]" operator="child_of"/>
                    <group expand="0" name="group_by" string="Group By">
-                       <filter name="salesperson" string="Salesperson" domain="[]" context="{'group_by' : 'user_id'}" />
-                       <filter string="Company" context="{'group_by': 'parent_id'}"/>
-                       <filter string="Country" context="{'group_by': 'country_id'}"/>
+                       <filter name="salesperson" string="Salesperson" context="{'group_by' : 'user_id'}" />
```

Comment:
why do you change that line?

Reply:
This filter's supposed to be for grouping. The empty `domain="[]"` attribute is not necessary, is it?

Reply:
always keep the diff minimal, please check our [stability guidelines](https://github.com/odoo/odoo/wiki/Contributing#what-does-stable-mean), we make no cleaning or improvement on stable versions